src/olympia/reviewers/tasks.py

```diff
@@ -22,3 +24,24 @@ def recalculate_post_review_weight(ids):
         for summary in summaries:
             summary.calculate_weight()
             summary.save()
+
+
+def get_recalc_needed_filters():
+    cv = '_current_version__'
```

Comment:
nit: IMHO the variable doesn't buy us anything, doesn't make it any easier to read the query parts below

Reply:
Indeed! It was useful for the first few versions of the query where I used it much much more, but then simplified the query more and more.

I'll remove it.

Reply:
Fixed in 6a6852231c